packages/backend/src/api/resources/Cookies.ts

```diff
@@ -0,0 +1,9 @@
+import type { CookiesJSON } from './JSON';
+
+export class Cookies {
+  constructor(readonly cookies: string[]) {}
+
+  static fromJSON(data: CookiesJSON): Cookies {
+    return new Cookies(data.cookies);
+  }
+}
```

Comment:
This shouldn't be needed anymore as per https://clerkinc.slack.com/archives/C07LKKV3M18/p1726128144736809. Instead, we return the standard (i.e. we use it on other places as well) token response